darwin/importer/importer.py

```diff
@@ -420,8 +502,8 @@ def _get_skeleton_name(skeleton: dt.AnnotationClass) -> str:
 
 
 def _handle_subs(
-    annotation: dt.Annotation, data: Dict[str, Any], annotation_class_id: str, attributes: Dict[str, Any]
-) -> Dict[str, Any]:
+    annotation: dt.Annotation, data: Dict[str, Unknown], annotation_class_id: str, attributes: Dict[str, Unknown]
+) -> Dict[str, Unknown]:
```

Comment:
Is this related to multiprocessing? Feels like it should be in a separate PR

Reply:
It's housekeeping, like the other one - I'm inclined to agree, but the nature of tasks needed on darwin-py so far has meant that the approach of fixing types a file at a time works well.  I can reverse it for readability - but the likelihood is that the ticket created to do it will not see light of day for a while.